### PR TITLE
feat: desktop - adds new featured buttons to portfolio screen

### PR DESCRIPTION
**Pull request by cjordan-ledger**
### 📝 Description

This adds components that will be used for the new exchange buttons on the LLD portfolio screen. This is currently hidden by a local dev feature flag.

### ❓ Context

- **Impacted projects**:  desktop
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-5174

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

#### Regular/hovered/disables states
image

#### Regular/hovered/disables states - light mode 
image

#### Tabbed state
image

#### With chevron
image

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/screens/dashboard/FeaturedButtons.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+const ButtonGrid = styled(Grid).attrs(() => ({
+  columns: 3,
+  columnGap: 4,
+}))`
+  margin-top: ${p => p.theme.space[4]}px;
+  margin-bottom: ${p => p.theme.space[6]}px;
+`;
+
+const devFeatureFlag = true;
```
Comment (sarneijim) [inline apps/ledger-live-desktop/src/renderer/screens/dashboard/FeaturedButtons.jsx:20]:
```suggestion
const devFeatureFlag = false;
```

File: apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  color: "neutral.c00",
+  borderRadius: 1,
+  translateY: "3px",
+  position: "relative",
+  top: "-1px",
+}))`
+  display: inline;
+`;
+
+const Label = ({ label }: { label: string }) => {
```
Comment (sarneijim) [inline apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx:106]: Can we use "tag" component for this?

File: apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
+// @flow
+
+import styled from "styled-components";
+import Text from "~/renderer/components/Text";
+import { focusedShadowStyle } from "~/renderer/components/Box/Tabbable";
+import Box from "~/renderer/components/Box";
+import React from "react";
+import ChevronRight from "~/renderer/icons/ChevronRight";
+
+const TitleText = styled(Text).attrs(() => ({
```
Comment (sarneijim) [inline apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx:10]: Can we use existing "typography"?

File: apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+  fontSize: 4,
+  fontWeight: "semiBold",
+  mb: "2px",
+}))`
+  transition: color ease-in-out 200ms;
+`;
+
+const BodyText = styled(Text).attrs(p => ({
```
Comment (sarneijim) [inline apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx:18]: Can we use existing "typography"?

**Review by sarneijim — CHANGES_REQUESTED** [review]

File: apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx (hunk 2 above)
Comment (cjordan-ledger) [reply apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx:106]: I don't think so as it looks quite different 
image

File: apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx (hunk 3 above)
Comment (sarneijim) [reply apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx:10]: https://react-ui-storybook.vercel.app/?path=/story/asorted-typography-text--overview

Comment (cjordan-ledger) [reply apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx:10]: I have been using this component `apps/ledger-live-desktop/src/renderer/components/Text.js` (which is used in lots of places in LLD) - and it doesn't seem to take in a `variant` prop. I'll try to use the Text component from react-ui instead.

File: apps/ledger-live-desktop/src/renderer/screens/dashboard/FeaturedButtons.jsx (hunk 1 above)
Comment (cjordan-ledger) [reply apps/ledger-live-desktop/src/renderer/screens/dashboard/FeaturedButtons.jsx]: Done https://github.com/LedgerHQ/ledger-live/pull/2396/commits/ac2356106ca762dddbabcc6c576bf9564e5b6f1d

File: apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx (hunk 3 above)
Comment (cjordan-ledger) [reply apps/ledger-live-desktop/src/renderer/components/EntryButton/EntryButton.jsx:10]: @sarneijim changed - now using `Text` from react-ui

**Review by sarneijim — APPROVED** [review]

**Review by brunospolidorio — APPROVED** [review]

**Pull request merged**